stubs/core.stub

```diff
@@ -69,7 +69,13 @@ function str_shuffle(string $string): string {}
 
 /**
  * @param array<mixed> $result
- * @param-out ($string is lowercase-string ? array<int|string, array<mixed>|lowercase-string> : array<int|string, array<mixed>|string>) $result
+ * @param-out ($string is lowercase-string
```

Comment:
Technically, a first check of 
`string is lowercase-string&uppercase-string` should be done since such character exists. (Like '42' or '-')

But I dunno if real case exists with `parse_str`.

Reply:
Fixed in 0c5e7213c